### DISCUSSION TITLE
parsers.lib.config

### DISCUSSION THREAD

**Discussion by EPRINC-MP** [electricitymaps/electricitymaps-contrib#4476] [Q&A] [answered]
Greetings, 

What python module is required for the following line:
from parsers.lib.config import refetch_frequency

I'm using Fedora 36 Linux; I can't seem to find it. 

Thank you.

**Comment by shuuji3** [comment, marked as answer]
Hello, @EPRINC-MP 🙂

The `parsers.lib.config` is not an external Python library, so you don't have to install an additional library for this import. It actually imports this Python file  https://github.com/electricitymap/electricitymap-contrib/blob/master/parsers/lib/config.py inside the project.

Hope this helps.
